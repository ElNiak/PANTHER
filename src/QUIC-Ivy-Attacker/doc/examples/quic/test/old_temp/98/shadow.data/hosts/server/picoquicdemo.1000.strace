00:00:00.000000000 [tid 1000] shadow_init_memory_manager(...) = 0
000000000000000000 [tid 1000] gettimeofday(...) = 0
000000000000000000 [tid 1000] time(...) = 946684800
00:00:00.000000000 [tid 1000] fstat(...) = 0
000000000000001020 [tid 1000] gettimeofday(...) = 0
00:00:00.000001020 [tid 1000] openat(-100, "/tmp/QUIC-FormalVerification/tickets/tic"..., (empty), (empty)) = 3
00:00:00.000002030 [tid 1000] fstat(...) = 0
00:00:00.000002030 [tid 1000] read(3, 0x555555618fc0, 4096) = 0
00:00:00.000004030 [tid 1000] close(3) = 0
00:00:00.000004030 [tid 1000] futex(...) = 0
00:00:00.000006030 [tid 1000] futex(...) = 0
00:00:00.000006030 [tid 1000] futex(...) = 0
00:00:00.000008030 [tid 1000] futex(...) = 0
00:00:00.000008030 [tid 1000] brk(0x555555653000) = 1432694784
00:00:00.000010030 [tid 1000] futex(...) = 0
00:00:00.000010030 [tid 1000] futex(...) = 0
00:00:00.000012030 [tid 1000] futex(...) = 0
00:00:00.000012030 [tid 1000] futex(...) = 0
00:00:00.000014030 [tid 1000] futex(...) = 0
00:00:00.000014030 [tid 1000] futex(...) = 0
00:00:00.000016030 [tid 1000] futex(...) = 0
00:00:00.000016030 [tid 1000] futex(...) = 0
00:00:00.000018030 [tid 1000] futex(...) = 0
00:00:00.000018030 [tid 1000] openat(-100, "/tmp/QUIC-FormalVerification/quic-implem"..., (empty), (empty)) = 3
00:00:00.000020030 [tid 1000] fstat(...) = 0
00:00:00.000020030 [tid 1000] read(3, 0x55555563be40, 4096) = 1159
00:00:00.000022030 [tid 1000] read(3, 0x55555563be40, 4096) = 0
00:00:00.000022030 [tid 1000] close(3) = 0
00:00:00.000024030 [tid 1000] openat(-100, "/tmp/QUIC-FormalVerification/quic-implem"..., (empty), (empty)) = 3
00:00:00.000024030 [tid 1000] fstat(...) = 0
00:00:00.000026030 [tid 1000] read(3, 0x55555563c100, 4096) = 1704
00:00:00.000026030 [tid 1000] close(3) = 0
00:00:00.000028030 [tid 1000] getrandom(0x7fffffffdf90, 32, 0) = 32
00:00:00.000028030 [tid 1000] getrandom(0x7fffffffe110, 24, 0) = 24
00:00:00.000030030 [tid 1000] getrandom(0x555555618d28, 64, 0) = 64
000000000000030030 [tid 1000] gettimeofday(...) = 0
00:00:00.000031040 [tid 1000] openat(-100, "demo_token_store.bin", (empty), (empty)) = -2 (ENOENT)
00:00:00.000031040 [tid 1000] write(2, "No token file present. Will create one a"..., 66) = 66
000000000000033040 [tid 1000] gettimeofday(...) = 0
00:00:00.000033040 [tid 1000] socket(Inet, 2, 17) = 3
00:00:00.000034050 [tid 1000] setsockopt(3, 0, 1, 0x7fffffffd714, 4) = 0
00:00:00.000034050 [tid 1000] setsockopt(3, 0, 13, 0x7fffffffd714, 4) = 0
00:00:00.000036050 [tid 1000] setsockopt(3, 0, 8, 0x7fffffffd734, 4) = 0
00:00:00.000036050 [tid 1000] bind(3, 11.0.0.2:4443, 16) = 0
00:00:00.000038050 [tid 1000] getsockname(3, 0x7fffffffd790, 128 (0x7fffffffd734)) = 0
00:00:00.000038050 [tid 1000] socket(Inet6, 2, 17) = -97 (EAFNOSUPPORT)
00:00:00.000040050 [tid 1000] write(1, "Starting Picoquic server (v1.01a) on por"..., 148) = 148
00:00:00.000040050 [tid 1000] exit_group(...) = <native>
