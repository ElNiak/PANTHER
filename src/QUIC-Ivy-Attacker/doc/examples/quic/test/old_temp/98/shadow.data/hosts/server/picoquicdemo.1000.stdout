Starting Picoquic server (v1.01a) on port 4443, server name = ::, just_once = 0, do_retry = 0
Server exit, ret = 0x41b
Server exit with code = 1051
